src/CampaignsAdmin/CampaignList.tsx

```diff
@@ -2,8 +2,8 @@ import React, { useCallback, useEffect, useState } from 'react';
 import { BrowserRouter as Router, Link } from 'react-router-dom';
 import { Table, Button, Popconfirm, Space } from 'antd';
 import { DeleteOutlined, EyeOutlined,EditOutlined } from '@ant-design/icons';
-import { getCampaigns, deleteCampaign } from '../db/Campaigns';
```

Comment:
This is the main change I wanted to get done on this PR. Everything else was little things I found along the way and fixed